sql/auto_uninstall.sql

```diff
@@ -1,23 +1,38 @@
-SET FOREIGN_KEY_CHECKS=0;
-
-DELETE FROM civicrm_setting WHERE `name` LIKE 'membershipextras_paymentplan_%';
```

Comment:
looks like the old uninstall script is overwritten.

Reply:
This is an example of what I said in my previous comment. These lines can't be deleted from the uninstallation script, so you'll need to add them in again.